egs/swbd/s5c/local/run_asr_segmentation.sh

```diff
@@ -1,15 +1,17 @@
-#! /bin/bash
+#!/bin/bash
 
-# Copyright 2017  Vimal Manohar
+# Copyright  2017  Nagendra Kumar Goel
+#            2017  Vimal Manohar
 # Apache 2.0
 
-# Features configs (Must match the features used to train the models
-# $sat_model_dir and $model_dir)
+# We assume the run-1-main.sh (because we are using model directories like
+# exp/tri4) and later we assumme run-4-anydecode.sh was run to prepare
+# data/dev10h.pem
```

Comment:
This comment looks like it is for a babel recipe, not switchboard, unless I am mistaken.

Reply:
fixed and pushed